lib/packages/ui/lib/src/widgets/media/image.dart

```diff
@@ -79,19 +82,15 @@ class _LMImageState extends State<LMChatImage> {
   }
 
   @override
-  void didUpdateWidget(covariant LMChatImage oldWidget) {
-    super.didUpdateWidget(oldWidget);
-    style = widget.style ?? LMChatTheme.theme.imageStyle;
+  void dispose() {
+    super.dispose();
```

Comment:
remove dispose() if we are not using it to dispose any other instance.

Reply:
Done